net/raft/raft.go

```diff
@@ -351,6 +357,28 @@ func (sv *Service) Join(bootURL string) error {
 	return nil
 }
 
+// Stop stops the Raft algorithm, stopping log replication. Once a
+// Service has been stopped, it's unusable. Stop must be called at
+// most once.
+func (sv *Service) Stop() error {
+	// signal the intention to stop
+	close(sv.stop)
+
+	// if the Service was never initialized, there are no
+	// goroutines to wait for and no open wal
+	if !sv.initialized() {
+		return nil
+	}
+
+	// wait for the service's goroutines to stop
+	sv.stopwg.Wait()
```

Comment:
ok, cool - can you add a comment somewhere (maybe on the stopwg field) that explains this?